app/models/experimentation/notifications_experiment.rb

```diff
@@ -66,6 +91,14 @@ def cancel
 
     private
 
+    def remaining_enrollments_allowed(date)
+      MAX_PATIENTS_PER_DAY - enrolled_patients(date).count
+    end
+
+    def enrolled_patients(date)
+      treatment_group_memberships.where(experiment_inclusion_date: date)
```

Comment:
This feels like an alias / delegation method, do we need it?

Reply:
On further thought, I think this highlights that "membership" and "enrolment" might actually mean the same thing. And we might want to just stick with one word instead of two.